src/Lean/Elab/Print.lean

```diff
@@ -116,7 +116,7 @@ private def printStructure (id : Name) (levelParams : List Name) (numParams : Na
           m := m ++ indentD (m!"{modifier}{.ofConstName proj (fullNames := true)}" ++ " : " ++ ftype)
       -- Constructor
       let cinfo := getStructureCtor (← getEnv) id
-      m := m ++ Format.line ++ "constructor:" ++ indentD (m!"{cinfo.name} : {cinfo.type}")
+      m := m ++ Format.line ++ "constructor:" ++ indentD (.signature cinfo.name)
```

Comment:
Can you apply this change to `inductive` too? I find it very annoying that `#print Lean.Expr` doesn't tell me what any of the constructor arguments are supposed to mean!

Reply:
I agree, but that will wait for another PR